openPrompt/idea_generator/readme.md

```diff
@@ -0,0 +1,62 @@
+## Idea Generator
+### Categories
+```
+['Generation']
```

Comment:
The categories format is different with others?

Reply:
will it be better to use inline code block for categories?
such as:
`category1`, `category2`